jboss-integration/src/main/java/org/teiid/jboss/rest/RestASMBasedWebArchiveBuilder.java

```diff
@@ -117,7 +119,280 @@ else if (securityType.equalsIgnoreCase("httpbasic")) {
 		return byteStream.toByteArray();
 	}
 
-	private void writeEntry(String name, ZipOutputStream out, byte[] contents) throws IOException {
+    private void writeSwagger(ZipOutputStream out, Properties props, VDBMetaData vdb) throws IOException {
```

Comment:
@rareddy 
> contents to the build somewhere,
replaceTemplates for all swagger static file is not necessary, we can't add all swagger directory via build, because the war generated dynamicly in deploying vdb, but we can recurse the swagger directory add each file as a zipEntry.